### DISCUSSION TITLE
Countdown component cannot deploy on vercel

### DISCUSSION THREAD

**Discussion by biskitsx** [saadeghi/daisyui#2391] [Q&A] [answered]
```
            <span className="countdown font-mono text-4xl">
                <span style={{ "--value": time }}></span>
            </span>
```
i use next typescript and its show 

Type error: Type '{ "--value": number; }' is not assignable to type 'Properties<string \| number, string & {}>'.
--
18:08:52.660 | Object literal may only specify known properties, and '"--value"' does not exist in type 'Properties<string \| number, string & {}>'.

**Comment by saadeghi** [comment, marked as answer]
This has nothing to do with Vercel or daisyUI.

Read this: https://curiouslychase.com/notes/fix-css-variable-keys-in-style-attributes-in-react-and-typescript/
> `style={{ "--value": string } as React.CSSProperties}`

If you're using React with Typescript, this is how you should use CSS variables.
